cfme/tests/cloud/test_cloud_proxy.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+# These tests don't work at the moment, due to the security_groups multi select not working
+# in selenium (the group is selected then immediately reset)
+import pytest
+
+import cfme.fixtures.pytest_selenium as sel
+from cfme.cloud.provider import CloudProvider
+from cfme.web_ui import flash, form_buttons
+from utils import testgen, error
+from utils.conf import cfme_data
+from utils.conf import credentials
+from utils.log import logger
+from utils.appliance.implementations.ui import navigate_to
+
+pytestmark = [pytest.mark.tier(1)]
```

Comment:
**Required**
Is there a requirement that applies to these tests? Need to mark it too